src/host/screenInfo.cpp

```diff
@@ -2521,13 +2521,11 @@ void SCREEN_INFORMATION::SetTerminalConnection(_In_ ITerminalOutputConnection* c
     OutputStateMachineEngine& engine = reinterpret_cast<OutputStateMachineEngine&>(_stateMachine->Engine());
     if (pTtyConnection)
     {
-        engine.SetTerminalConnection(pTtyConnection,
-                                     std::bind(&StateMachine::FlushToTerminal, _stateMachine.get()));
+        engine.EnableFlushing(std::bind(&StateMachine::FlushToTerminal, _stateMachine.get()));

```

Comment:
I wonder if we should file low-prio followup work to clear up the ownership here: the state machine engine shouldn't need a function bound into it from the state machine itself, especially not by an unrelated class.